### ISSUE TITLE
Move to a src layout

### ISSUE THREAD

**Issue by MatthieuDartiailh** [MatthieuDartiailh/oculy#2]
https://github.com/MatthieuDartiailh/bytecode/pull/99 provide a good example of such a move.

**Comment by valizadehm** [comment]
I'm going to address this issue. Although it seems a simple move of *oculy* directory into *src* directory, it needs to adjust the relative dirctories in `pyproject.toml`, I guess.

**Comment by MatthieuDartiailh** [comment]
Since we got rid of the where clause it should work out of the box the src organization is understood by default by setuptools.

**Comment by valizadehm** [comment]
You mean that we no longer need to add what it follows in our *pyproject.toml* file?
````
[options]
package_dir=
    =src

[options.packages.find]
where=src
```

**Comment by MatthieuDartiailh** [comment]
No those are not needed.

**Comment by valizadehm** [comment]
You're absolutely right. Running in developping mode `pip install e .`, the package was built successfully after I moved *oculy* folder into *src* folder.

```
Successfully installed oculy-0.1.dev51+g87f5d3b.d20220901
```

**Comment by valizadehm** [comment]
But changing the directory, there are 57 files as well as 57 unversioned files to commit. In your opinion, is it ok?

**Comment by MatthieuDartiailh** [comment]
You need to commit wverything at once and git will figure it is just file moving around.

**Comment by valizadehm** [comment]
> You need to commit wverything at once and git will figure it is just file moving around.

👍

**Comment by MatthieuDartiailh** [comment]
Closed by #14
